e2e/playwright/editor-tests.spec.ts

```diff
@@ -1206,4 +1206,42 @@ test.describe('Editor tests', { tag: ['@skipWin'] }, () => {
       })
     }
   )
+
+  test('Can select lines on the main axis', async ({ page, homePage }) => {
+    await page.addInitScript(async () => {
+      localStorage.setItem(
+        'persistCode',
+        `sketch001 = startSketchOn(XZ)
+  profile001 = startProfileAt([100.00, 100.0], sketch001)
+    |> yLine(length = -100.0)
+    |> xLine(length = 200.0)
+    |> line(endAbsolute = [profileStartX(%), profileStartY(%)])
+    |> close()`
+      )
+    })
+
+    const width = 1200
+    const height = 800
+    const viewportSize = { width, height }
+    await page.setBodyDimensions(viewportSize)
+
+    await homePage.goToModelingScene()
+
+    const u = await getUtils(page)
+    await u.waitForPageLoad()
+
+    // open sketch
+    await page.mouse.dblclick(width / 2, height / 2)
```

Comment:
I think this could be the flakiness problem. `waitForPageToLoad` doesn't guarantee that the scene is rendered and ready to be double-clicked on I believe. We have `scene.settled` from our `scene` fixture that could be more reliable, but I think since the sketch entrance is not the system under test you should use `toolbar.editSketch(0)` which uses the feature tree and is rock solid in my testing.

Reply:
Yeah that's an acceptable level, thanks for making the change.